components/infobox/wikis/starcraft/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,510 @@
+
+	local currentYearEarnings = _earningsGlobal[tostring(_CURRENT_YEAR)]
+	if currentYearEarnings then
+		currentYearEarnings = Math.round{currentYearEarnings}
+		currentYearEarnings = '$' .. mw.language.new('en'):formatNum(currentYearEarnings)
+	end
+
+	return {
+		Cell{
+			name = 'Approx. Winnings ' .. _CURRENT_YEAR,
+			content = {currentYearEarnings}
+		},
+		--Cell{name = 'Years active', content = {yearsActive}}
+	}
+end
+
+function CustomPlayer.nameDisplay()
+	CustomPlayer._getRaceData(_args.race or 'unknown')
+	local raceIcon = RaceIcon.getNormalIcon{_raceData.race}
+	local name = _args.id or _player.pagename
+
+	return raceIcon .. '&nbsp;' .. name
+end
+
+function CustomPlayer._getRaceData(race, asCategory)
+	race = string.lower(race)
+	race = CleanRace[race] or race
+	local raceTable = _RACE_DATA[race]
+
+	local faction, faction2
+	if race == _RACE_ALL_SHORT then
+		faction = _RACE_ALL
+	else
+		faction = (raceTable or {})[1]
+		faction2 = (raceTable or {})[2]
+	end
+
+	local display
+	if not raceTable and race ~= 'unknown' then
+		display = '[[Category:InfoboxRaceError]]<strong class="error">' ..
+			mw.text.nowiki('Error: Invalid Race') .. '</strong>'
+	else
+		if asCategory then
+			for raceIndex, raceValue in ipairs(raceTable or {}) do
+				raceTable[raceIndex] = ':Category:' .. raceValue .. ' Players|' .. raceValue .. ']]'
+					.. '[[Category:' .. raceValue .. ' Players'
+			end
+		end
+		if raceTable then
+			display = '[[' .. table.concat(raceTable, ']],&nbsp;[[') .. ']]'
+		end
+	end
+
+	_raceData = {
+		race = race,
+		faction = faction or '',
+		faction2 = faction2 or '',
+		display = display,
+	}
+
+	return display
+end
+
+function CustomPlayer.adjustLPDB(_, lpdbData)
+	_lpdbData = lpdbData
+	local extradata = lpdbData.extradata or {}
+	extradata.race = _raceData.race
+	extradata.faction = _raceData.faction
+	extradata.faction2 = _raceData.faction2
+	extradata.lc_id = string.lower(_player.pagename)
+	extradata.teamname = _args.team
+	extradata.activeplayer = (not _statusStore) and Variables.varDefault('isActive', '') or ''
+
+	if Variables.varDefault('racecount') then
+		extradata.racehistorical = true
+		extradata.factionhistorical = true
+	end
+
+	--Notability values per year
+	for year = Info.startYear, _CURRENT_YEAR do
+		extradata['notabilityin' .. year] = Notability.notabilityScore{
+			players = _player.pagename,
+			startdate = year .. _FIRST_DAY_OF_YEAR,
+			enddate = year .. _LAST_DAY_OF_YEAR,
+			smmult = 0.5,
+		}
+	end
+
+	lpdbData.extradata = extradata
+
+	return lpdbData
+end
+
+function CustomPlayer.getStatusToStore()
+	if String.isNotEmpty(_args.status) then
+		_statusStore = mw.getContentLanguage():ucfirst(_args.status)
+	elseif _args.death_date then
+		_statusStore = 'Deceased'
+	elseif _args.retired then
+		_statusStore = 'Retired'
+	elseif
+		(not Logic.readBool(_args.isplayer)) and
+		string.lower(_args.role or _args.defaultPersonType) ~= 'player'
+	then
+		_statusStore = 'not player'
```

Comment:
How much work would it be moving to `type` instead of `status` for this?

Reply:
if you mean replacing `_statusStore` with e.g. `_type` or soemthing like that (i.e. variable rename) then easy as one can just replace it
else i don't get the meaning of the comment^^

Reply:
Saw you removed `not player` entirely. Don't need that data?

Reply:
they strictly sep. players and commentators and do not have any other types
and they do not have any role support anyways
so it was redundant